package.json

```diff
@@ -29,6 +29,7 @@
   },
   "homepage": "https://github.com/christopherthielen/check-peer-dependencies#readme",
   "dependencies": {
+    "launchdarkly-node-client-sdk": "^2.1.0",
```

Comment:
I do not think feature flags should be part of this package. This library's job is to check for peer dependencies. the feature flag of whether it checks for peer dependency or not belongs to the caller of this package, not encoded within the package itself.